django/contrib/admindocs/utils.py

```diff
@@ -78,11 +79,13 @@ def parse_rst(text, default_reference_context, thing_being_parsed=None):
 
 .. default-role::
 """
+    # Ensure compatibility with docutils < 0.22 and also 0.22+.
+    writer_instance = docutils.writers.get_writer_class("html")()
     parts = docutils.core.publish_parts(
         source % text,
         source_path=thing_being_parsed,
         destination_path=None,
-        writer="html",
+        writer=writer_instance,
```

Comment:
I'd consider adding release notes (`5.2.5.txt`):
```
* Added compatibility for ``docutils`` 0.22 (:ticket:`36535`)
```

Reply:
Just a note that we'd need to add the release note on main as well :+1:
Unsure if it's easier to target main, backport, then do the drop of docutils < 0.22 support

Reply:
@felixxm but if we add it only to 5.2.5 there would be a discrepancy between `main`  and `stable/5.2.x`... I haven't seen that before! (other than the version bumps for releasing).